src/transformers/models/sam/processing_tf_sam.py

```diff
@@ -0,0 +1,248 @@
+# coding=utf-8
```

Comment:
What is the purpose of this file?

Reply:
Shh, it's gone now. We don't talk about `processing_tf_sam`